src/main/java/org/jenkinsci/plugins/github_branch_source/PullRequestSCMHead.java

```diff
@@ -106,7 +106,7 @@ private Object readResolve() {
                     merge,
                     metadata.getNumber(),
                     new BranchSCMHead(metadata.getBaseRef()),
-                    metadata.getUserLogin(),
+                    null,
```

Comment:
@stephenc would it have any side effect, assuming this is how the code was before?